ci/code_checks.sh

```diff
@@ -80,6 +80,13 @@ if [[ -z "$CHECK" || "$CHECK" == "lint" ]]; then
     flake8-rst doc/source --filename=*.rst --format="$FLAKE8_FORMAT"
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
+    # Check if pandas is referenced as pandas, not *pandas* or Pandas
+    MSG='Checking if pandas reference is standardized or not' ; echo  $MSG
+    grep -nr '*pandas*|Pandas' doc/*
+    grep -nr '*pandas*|Pandas' web/*
+    RET=$(($RET + $?)) ; echo $MSG "DONE"
+
+
```

Comment:
Don't leave extra blank lines, keep the same style and formatting as the rest of the script.